http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -54,7 +54,27 @@ import scala.concurrent.duration._
 import scala.jdk.CollectionConverters._
 
 object Config {
-  val config = ConfigFactory.parseResources("leonardo.conf").withFallback(ConfigFactory.load()).resolve()
+
+  /** Loads all the configs for the Leo App. All values defined in `src/main/resources/leo.conf` will take precedence over any other configs. In this way, we
+   * can still use configs rendered by `firecloud-develop` that render to `config/leonardo.conf` if we want.
+   *     If you want to use leo.conf you must populate the appropriate ENV vars (done in terra-helmfile)
+   *     leonardo.conf is what firecloud-develop renders, and it will be used if the ENV vars are not populated for leo.conf
+   */
+  // Config that lives in leo repo in /src/main/resources
+  val leoConfig = ConfigFactory.parseResourcesAnySyntax("leo")
+
+  // Config that is generated to /config from firecloud-develop
+  val backupConfig1 = ConfigFactory
+    .parseResources("leonardo.conf")
+
+  // Load any other configs on the classpath following: https://github.com/lightbend/config#standard-behavior
+  // This is where things like `src/main/resources/reference.conf` will get loaded
+  val backupConfig2 = ConfigFactory.load()
+
+  val config = leoConfig
```

Comment:
would love a comment here on which config has priority to overwrite at a glance.